src/libmscore/score.cpp

```diff
@@ -2750,6 +2750,46 @@ void Score::adjustKeySigs(int sidx, int eidx, KeyList km)
     }
 }
 
+//---------------------------------------------------------
+//   getKeyList
+//      This is taken from MuseScore::editInstrList()
+//---------------------------------------------------------
+
+KeyList Score::getKeyList() const
+{
+    // find the keylist of the first pitched staff
+    KeyList tmpKeymap;
```

Comment:
If it's a KeyList does it make sense to call the variable "tmpKeymap"?  Even "tmp" doesn't seem quite right either, given it's the returned result of the function (even if that's by creating a copy).

Reply:
True, but as the comment said, it is just copied from 3.x code. Changing too much makes it unclear to track where this code original came from.